src/command.js

```diff
@@ -23,55 +23,80 @@ export function matchImageSnapshotCommand(defaultOptions) {
 
     const startTime = Date.now();
 
+    let cypressContext = Cypress.mocha.getRunner().suite.ctx.test;
+    let cypressPath = [cypressContext.title];
+    let rootCypressContext = cypressContext;
+    while (rootCypressContext.parent != null) {
+      rootCypressContext = rootCypressContext.parent;
+      cypressPath.push(rootCypressContext.title);
+    }
+
+    var name = typeof maybeName === 'string' ? maybeName : undefined;
+    const rawSnapshotFilepath = cypressContext.titlePath();
```

Comment:
In the doc https://mochajs.org/api/suite#titlePath, `titlePath` returns a single string and not an array. Are you sure it's an array and we always need the top of it?

Reply:
It seems to be an array in cypress tests. We need the full array to give meaningful names to the folders for the snapshots.